components/brave_news/browser/resources/FeedNavigation.tsx

```diff
@@ -153,9 +153,8 @@ export default function Sidebar() {
       <summary>
         {subscribedChannels.length ? Marker : PlaceholderMarker}
         {getLocale('braveNewsChannelsHeader')}
-        <AddButton size="tiny" onClick={e => {
```

Comment:
Svelte events don't propagate, so this was doing nothing and getting broken by the new types